account_payment_term_partner_holiday/models/res_partner.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2021 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    holiday_ids = fields.One2many(
+        comodel_name="res.partner.holiday",
+        inverse_name="partner_id",
+        string="Holidays",
+        copy=True,
+        auto_join=True
+    )
+
+    def is_date_in_holiday_ids(self, date):
+        if isinstance(date, str):
```

Comment:
This is not happening in v12 AFAIK, so no need to add this condition. Anyways, it should be `fields.Date`, not `fields.Datetime`.